src/sql/workbench/parts/notebook/common/models/notebookModel.ts

```diff
@@ -977,7 +985,7 @@ export class NotebookModel extends Disposable implements INotebookModel {
 	/**
 	 * Serialize the model to JSON.
 	 */
-	toJSON(): nb.INotebookContents {
+	toJSON(type?: NotebookChangeType): nb.INotebookContents {
```

Comment:
>type [](start = 8, length = 4)

Take in the type but never actually do anything with it? Shouldn't you pass it to c.toJSON()?